openpype/hosts/traypublisher/plugins/publish/validate_colorspace.py

```diff
@@ -33,13 +34,25 @@ def process(self, instance):
             config_path = colorspace_data["config"]["path"]
             if config_path not in config_colorspaces:
                 colorspaces = get_ocio_config_colorspaces(config_path)
-                config_colorspaces[config_path] = set(colorspaces)
+                if not colorspaces.get("colorspaces"):
+                    raise PublishValidationError(
+                        title="Colorspace validation",
+                        message=f"OCIO config '{config_path}' does not contain "  # noqa
+                                f"any colorspaces. This is error in config. "
+                                "Contact your pipeline TD.",
+                        description=f"OCIO config '{config_path}' does not "
+                                    f"contain any colorspaces. This is error "
+                                    "in config. Contact your pipeline TD."
+                    )
```

Comment:
As far as I can see the message and description is the same. Define the string once, re-use for both.

Also, this is artist-facing so let's fix up the grammar.

```suggestion
                    message = (
                        f"OCIO config '{config_path}' does not contain any "
                        "colorspaces. This is an error in the OCIO config. "
                        "Contact your pipeline TD.",
                    )
                    raise PublishValidationError(
                        title="Colorspace validation",
                        message=message,
                        description=message
                    )
```